src/casp/scripts/README.rst

```diff
@@ -0,0 +1,13 @@
+Scripts
+-------
+
+This package includes scripts that are necessary for keeping the infrastructure. Such scripts are used to build
+resource files used by backend application, generating data extracts, running benchmarking, etc.
```

Comment:
@KevinCLydon yeah, I agree this text is unclear. Do you think this would be better?

"
This package includes scripts needed to maintain the services and data in the CAS backend application. These scripts are used to build resource files (e.g., a resource file for cell type ontology used by the Consensus Engine), generate data extracts, run benchmarks, and more. They are not exposed for execution by CAS users.

Reply:
That sounds perfect!